doc/authentication-methods/client-certificate.md

```diff
@@ -33,8 +33,16 @@ The `SASL EXTERNAL` authentication method requires a digital client certificate.
 This digital certificate should contain `xmppAddr` field(s), which is always checked first.
 If there is more than one JID specified in the `xmppAddr` fields, the client must include the authorisation entity which corresponds to the one of the specified JIDs.
 
-When no `xmppAddr` is specified, the `cn` (common name) field might be used to provide client's username, but it is optional (enabled by default).
-Usage of a `cn` field can be disabled by adding the `{authenticate_with_cn, false}` tuple to the list of `auth_opts` in MongooseIM config file.
+When no `xmppAddr` is specified, the `cn` (common name) field might be used to provide client's username, but it is optional (not enabled by default).
+There are three possible ways of using `SASL EXTERNAL` authentication
+method. It can be configured by adding one of the following options
```

Comment:
It can be configured by adding one of the following options to the list of `auth_opts` in MongooseIM config file: